### PR TITLE
Create subcontractors-paid-10-per-hour.en.md

### PR DESCRIPTION
**Pull request by realjenny**

### CODE REVIEW

**Review by stephenmcmurtry — APPROVED** [review]

File: frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{{< header />}}
+
+_Press contact: press@alphabetworkersunion.org_
+
+(Mountain View, CA) —In 2019, after sustained internal pressure from workers, Alphabet publicly announced “that vendor and staffing partners working with Google and
+other Alphabet subsidiaries in the U.S. will be required to provide a new benchmark of benefits for their workers, including a $15/hour minimum wage, 12 weeks of paid
+family leave, eight days of paid sick leave, $5000/year in tuition reimbursement, and comprehensive healthcare.” Google did not clarify that their own internal policies
+would prevent hundreds of workers from accessing these benefits. Through the use of various loopholes, including Google’s refusal to allow certain workers access to
+@google.com work email addresses, many workers are paid as little as $10/hour. In particular, workers at RaterLabs, who provide critical feedback to Google’s AI in
+order to improve the effectiveness of search rankings and ads on Google Search, Youtube, Google pages and more, are denied access to the claimed $15/hr pay, paid
```
Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:18]: s/Youtube/YouTube (if we care?)

File: frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+family leave, paid sick leave, health benefits & tuition reimbursements.
```
Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:19]: s/benefits & tuition/benefits, and tuition

File: frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+@google.com work email addresses, many workers are paid as little as $10/hour. In particular, workers at RaterLabs, who provide critical feedback to Google’s AI in
+order to improve the effectiveness of search rankings and ads on Google Search, Youtube, Google pages and more, are denied access to the claimed $15/hr pay, paid
+family leave, paid sick leave, health benefits & tuition reimbursements. 
+
+“It is unacceptable that Google would claim to be setting a new standard for contract workers with a $15 dollar an hour minimum, while knowingly establishing obvious
+loopholes that allow both Google and subcontractors to exploit workers. At a company valued at hundreds of billions of dollars, no worker should be making poverty
+wages. Not only is the $15 dollar an hour wage not a dignified wage—it is a minimum made inaccessible to critical workers. Google must work directly with impacted
+workers to establish a new minimum for all workers and end the loopholes that deny hard working people their fair share,” said Christopher Colley, RaterLabs worker
```
Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:24]: s/hard working/hardworking

File: frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+and member of the Alphabet Workers Union-CWA. 
+
+Workers who are [exempt from these benefits](https://support.google.com/corporate-suppliers/answer/10208902?hl=en#zippy=%2Cbenefits-eligibility) include workers who
+work less than 30 hours a week, workers who do not have an access badge, and workers who do not have an @google.com work email address. Workers at RaterLabs are not
+supplied with a @google.com email address, and are capped at a maximum of 26 hours a week. Thus, RaterLabs workers do not qualify for Google’s publicized benefits for
```
Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:29]: s/with a @/with an @

File: frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+temporary, vendor, and contract workers. Instead, RaterLabs workers earn slightly above the state minimum wage, with pay as low as $10/hr, even as they work to improve Google's most successful products. For hundreds of workers, including those at RaterLabs, Google’s claimed minimum benefits do not apply. 
+
+Google also invites workers to reach out to learn more about what benefits they should receive, and to
+[file complaints about mistreatment at work](https://about.google/intl/ALL_us/supplier-code-of-conduct/). Yet workers are not given direct access to the resources
+needed to learn more about expected benefits or the process for filing complaints. 
+
+“Google is not an equitable workplace. Temporary, vendor and contract workers who are explicitly denied access to the same pay, benefits and resources made available
```
Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:36]: s/vendor and/vendor, and

Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:36]: s/workers who are explicitly/workers are explicitly

Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:36]: s/benefits and resources/benefits, and resources

File: frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+to full time workers—even when they complete the same work as employees. We have heard of workers forced to work while in their hospital bed after childbirth because
```
Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:37]: s/full time/full-time

Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:37]: this sentence makes it seem like it's juxtaposing full time workers and employees - needs to be clarified that these are the same groups.

I suggest "to full-time workers--even when they complete the same work as said full-time employees" (leaving the em dash, I just don't know how to type it)

File: frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+they do not have access to paid sick leave. Other workers are working 26 hours a week while balancing a full student course load because they are denied tuition
+reimbursements. This is unacceptable. Google must immediately end all loopholes that deny workers access to the minimum benefits they have so loudly published without
+full implementation,” said Ashok Chandwaney, a software engineer at Google and member of the Alphabet Workers Union.
+
+The Alphabet Workers Union-CWA invites
+[any Alphabet contractor who has been denied access to Google’s publicized benefits to reach out](mailto:contact@alphabetworkersunion.org) and calls on Google to
```
Comment (junefish) [inline frontend/content/press/releases/subcontractors-paid-10-per-hour.en.md:43]: to make this more screen reader friendly, I think the linked text should just be "reach out".  That way when someone is skipping from link to link they will know that's the link to reach out without having to listen to the full text

**Pull request merged**